R/main.R

```diff
@@ -1,10 +1,19 @@
-#' shiny.i18n: A package for Shiny applications internationalization.
+#' i18n
 #'
-#' The shiny.i18n package provides functions for easy Shiny applications internationalization.#'
+#' Returns a translator object which can be used for translations.
 #'
-#' @section Foo functions:
-#' The foo functions ...
 #'
-#' @docType package
-#' @name shiny.i18n
-NULL
+#' @param translation_file character path to translation file
+#' @param key_translation character with language code, e.g. "en", "pl".
+#' If NULL, two first characters of the LANG env variable will be used.
+#'
+#' @return Translator object
+#' @export
+#'
+#' @examples
+#' #TODO
+i18n <- function(translation_file, key_translation = NULL) {
```

Comment:
hmm do you think this function adds that much value? maybe we don't need it at all? probably taking the language from $LANG doesn't have much value, since this is depends on whatever is set on the host -- most likely everyone will have their own way of managing the language to be set

Reply:
I believe yes. Taking `key_translation` from `$LANG` is optional (if argument is empty). The second reason for this function is that RC are still not that common in R, so many users can have difficulties with calling constructors. The aim of this wrapper was to help them.